webshare/share-url-invalid.https.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Share with an invalid URL</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+  </head>
+  <body>
+    <script>
+        promise_test(t => {
+          // URL is invalid in that the URL Parser returns failure (port is too
+          // large).
+          const url = 'http://example.com:65536';
+          return promise_rejects(
+              t, new DOMException(undefined, 'TypeError'),
```

Comment:
I'm getting this error on Safari (according to bots):

    function is not a constructor (evaluating 'new DOMException(undefined, 'SecurityError')')

How are you supposed to check for a particular exception type in a way that works in Safari?